packages/react-charting/jest.config.js

```diff
@@ -23,6 +23,7 @@ const config = createConfig({
   moduleNameMapper: {
     ...getEsmOnlyPackagesToCjsMapping(),
   },
+  modulePathIgnorePatterns: ['<rootDir>/src/utilities/testData.test.ts'],
```

Comment:
why this change

Reply:
because otherwise running the tests also expects a test to be present within the test file

Reply:
then remove the test keyword and add entry in npmignore

Reply:
I used following pattern https://github.com/microsoft/fluentui/pull/30117/files#diff-5e34a7bb2b3ed4a1697b714cd896f3587ab5f081495dd722712d658f6a4c55ab

this will also mitigate need of change files (as nothing related to implementation is changed)